Podfile

```diff
@@ -3,7 +3,12 @@ use_frameworks!
 
 raise 'Please use bundle exec to run the pod command' unless defined?(Bundler)
 
+def common_pods
+  pod 'SwiftLint', '~> 0.25'
+end
+
 target 'swiftgen' do
+  common_pods
   pod 'Commander', '~> 0.8'
   pod 'StencilSwiftKit', '~> 2.3'
   podspec :path => 'SwiftGenKit.podspec'
```

Comment:
Same notes as #401: maybe that could be moved to `common_pods` now?

Reply:
Note: this is a change from #401, so it'll disappear from this PR once that one is merged.